Alignment/OfflineValidation/test/zmumuValidate.ini

```diff
@@ -0,0 +1,228 @@
+#condition SiPixelTemplateDBObjectRcd = frontier://FrontierProd/CMS_CONDITIONS,SiPixelTemplateDBObject_38T_2015_v1_hltvalidation
+condition SiPixelTemplateDBObjectRcd = sqlite_file:/afs/cern.ch/user/d/dkotlins/public/CMSSW/DB/310815/SiPixelTemplateDBObject_38T_2015_v3.db,SiPixelTemplateDBObject38Tv10
+color = 633
+style = 1
+
+# No Zmumu
+[alignment:alignment_mp1834p1]
+title=alignment_mp1834p1
+globaltag = GR_P_V56
+#condition TrackerAlignmentErrorExtendedRcd = zeroAPE
+condition TrackerAlignmentRcd = sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/MP/MPproduction/mp1834/jobData/jobm1/alignments_MP.db, Alignments
+#condition TrackerSurfaceDeformationRcd = sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/MP/MPproduction/mp1834/jobData/jobm1/alignments_MP.db, Deformations
+condition SiStripBackPlaneCorrectionRcd = frontier://FrontierProd/CMS_CONDITIONS,SiStripBackPlaneCorrection_deco_GR10_v4_offline,deconvolution
+condition SiStripLorentzAngleRcd = frontier://FrontierProd/CMS_CONDITIONS,SiStripLorentzAngleDeco_v3_offline,deconvolution
+condition SiPixelLorentzAngleRcd  = frontier://FrontierProd/CMS_CONDITIONS,SiPixelLorentzAngle_2015_v2_hltvalidation
+#condition SiPixelTemplateDBObjectRcd = frontier://FrontierProd/CMS_CONDITIONS,SiPixelTemplateDBObject_38T_2015_v1_hltvalidation
+condition SiPixelTemplateDBObjectRcd = sqlite_file:/afs/cern.ch/user/d/dkotlins/public/CMSSW/DB/310815/SiPixelTemplateDBObject_38T_2015_v3.db,SiPixelTemplateDBObject38Tv10
+color = 633
+style = 1
+
+
+###############################################################################
+# configuration of individual validations
+
+[zmumu:Run2015B_ALCARECO_251604-251642]
+etamaxneg = 2.4
+etaminneg = -2.4
+etamaxpos = 2.4
+etaminpos = -2.4
+maxevents = -1
+firstRun=251604
+lastRun=251642
+dataset = /DoubleMuon/Run2015B-TkAlZMuMu-PromptReco-v1/ALCARECO
+
+
+###############################################################################
+# configure which validation to run on which alignment
+
+[validation]
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_defgt
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_PCL
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_hp1394
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_hp1398
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_hp1400
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_hp1401
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1819
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1820
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1823p4
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1826p1
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1826p2
+zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1834p0
+zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1834p1
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1835p0
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1835p1
```

Comment:
I do not know it is good to have this "legacy" ini file in release. As you can see, there are many old conditions.